### PR TITLE
Add EEPs link

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
This PR adds the link to EEP documents.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -22,6 +22,9 @@ <h2>Development Guides</h2>
   <ul>
     <li><a href="https://docs.google.com/document/d/1oKpvgstKlgmgUUja8hYqTqWxtwsgIbONoUaEj8lO0FE/edit">Embulk Input Plugin Development Guide</a></li>
   </ul>
+  <ul>
+    <li><a href="https://github.com/embulk/embulk/blob/master/docs/eeps/eep-0000.md">EEPs(Embulk Enhancement Proposals)</a></li>
+  </ul>
```
Comment (dmikurube) [inline docs/index.html:27]: * You may want to add this `<li>` block within the `<ul>` block above.
* Let's change the description to `Embulk Enhancement Proposals (EEPs)`

**Review by dmikurube — COMMENTED** [review]
Thanks. Left a comment.

File: docs/index.html (hunk 1 above)
Comment (hiroyuki-sato) [reply docs/index.html:27]: Thanks! Fixed.

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**